.php_cs.cache

```diff
@@ -0,0 +1 @@
+{"php":"7.4.2","version":"2.16.1:v2.16.1#c8afb599858876e95e8ebfcd97812d383fa23f02","indent":"    ","lineEnding":"\n","rules":{"visibility_required":true,"ternary_to_null_coalescing":true,"encoding":true,"full_opening_tag":true,"blank_line_after_namespace":true,"braces":true,"class_definition":true,"constant_case":true,"elseif":true,"function_declaration":true,"indentation_type":true,"line_ending":true,"lowercase_keywords":true,"method_argument_space":{"on_multiline":"ensure_fully_multiline"},"no_break_comment":true,"no_closing_tag":true,"no_spaces_after_function_name":true,"no_spaces_inside_parenthesis":true,"no_trailing_whitespace":true,"no_trailing_whitespace_in_comment":true,"single_blank_line_at_eof":true,"single_class_element_per_statement":{"elements":["property"]},"single_import_per_statement":true,"single_line_after_imports":true,"switch_case_semicolon_to_colon":true,"switch_case_space":true,"array_indentation":true,"array_syntax":{"syntax":"short"},"align_multiline_comment":{"comment_type":"all_multiline"},"binary_operator_spaces":true,"blank_line_after_opening_tag":true,"blank_line_before_statement":true,"cast_spaces":true,"class_attributes_separation":true,"combine_consecutive_issets":true,"combine_consecutive_unsets":true,"compact_nullable_typehint":true,"concat_space":{"spacing":"one"},"PedroTroller\/exceptions_punctuation":true,"PedroTroller\/line_break_between_method_arguments":{"max-args":10},"PedroTroller\/comment_line_to_phpdoc_block":true,"PedroTroller\/useless_code_after_return":true},"hashes":{"src\/Knp\/Snappy\/AbstractGenerator.php":3020002203,"src\/Knp\/Snappy\/GeneratorInterface.php":3708897097,"src\/Knp\/Snappy\/Image.php":207790329,"src\/Knp\/Snappy\/Pdf.php":2353233392,"src\/Knp\/Snappy\/Exception\/FileAlreadyExistsException.php":1655581840,"tests\/Knp\/Snappy\/AbstractGeneratorTest.php":2344254468,"tests\/Knp\/Snappy\/PdfTest.php":156588972,"tests\/Knp\/Snappy\/ImageTest.php":4066889974}}
```

Comment:
I’m not sure it’s a good idea to commit cache files to the repo :wink: Maybe you can remove it, and add a `*.cache` rule to the `.gitignore` file in order to avoid this?

Reply:
It's not related to this PR, I'm adding the CSFixer on a different PR and it has already been added on .gitignore there :wink:
It has erroneously been added during a branch switch :disappointed: